packages/vscode-webui/src/components/tool-invocation/tools/start-background-job.tsx

```diff
@@ -37,6 +47,16 @@ export const StartBackgroundJobTool: React.FC<
       detail={
         backgroundJobId ? (
           <BackgroundJobPanel backgroundJobId={backgroundJobId} />
+        ) : command ? (
+          <CommandPanelContainer
+            icon={
+              <div className="flex size-[16px] items-center justify-between">
```

Comment:
https://github.com/TabbyML/pochi/blob/2880e21d5fb771b6817ed63b47f0b0bf782e324a/packages/vscode-webui/src/components/tool-invocation/command-execution-panel.tsx#L282

keep the same style as execute command